images/cassandra/rootfs/run.sh

```diff
@@ -24,8 +24,9 @@ CASSANDRA_LISTEN_ADDRESS=${POD_IP}
 CASSANDRA_BROADCAST_ADDRESS=${POD_IP}
 CASSANDRA_BROADCAST_RPC_ADDRESS=${POD_IP}
 
-# Turn off JMX auth
-CASSANDRA_OPEN_JMX="${CASSANDRA_OPEN_JMX:-false}"
+# Turn on JMX auth
+CASSANDRA_OPEN_JMX=true
```

Comment:
this variable is set to `true` in https://github.com/gravitational/pithos-app/blob/master/resources/cassandra.yaml#L47